site/_docs/reference.md

```diff
@@ -2783,7 +2783,10 @@ In the following:
 | b f s | LENGTH(string)                             | Equivalent to `CHAR_LENGTH(string)`
 | h s | LEVENSHTEIN(string1, string2)                | Returns the Levenshtein distance between *string1* and *string2*
 | b | LOG(numeric1 [, numeric2 ])                    | Returns the logarithm of *numeric1* to base *numeric2*, or base e if *numeric2* is not present
+| m s | LN(numeric)                                  | Returns the natural logarithm (base *e*) of *numeric*
```

Comment:
`LN` and `LOG10` are defined in both built-in and engine extension functions, which may confuse users.

Reply:
I don’t think so. Users can still choose the log10 and ln functions in SqlStdOperatorTable.